esmvalcore/_task.py

```diff
@@ -593,6 +648,92 @@ def __str__(self):
         return txt
 
 
+def _get_response(url):
+    """Return information from CMIP6 Data Citation service in json format."""
+    json_data = False
+    if url.lower().startswith('https'):
+        try:
+            response = requests.get(url)
+            if response.status_code == 200:
+                json_data = response.json()
+            else:
+                logger.info('Error in the CMIP json link')
+        except IOError:
+            logger.info('Error in receiving the CMIP json file')
+    return json_data
+
+
+def _valid_json_data(data):
+    valid_data = False
+    keys = ['identifier', 'creators', 'titles', 'publisher', 'publicationYear']
+    if all(key in data for key in keys):
+        check_names = all('creatorName' in item for item in data['creators'])
+        if 'id' in data['identifier'] and check_names:
+            valid_data = True
+    return valid_data
+
+
+def _json_to_bibtex(data):
+    """Make a bibtex entry from CMIP6 Data Citation json data."""
+    url = ''.join(['https://doi.org/', data['identifier']['id']])
+    author_list = []
+    for item in data['creators']:
+        author_list.append(item['creatorName'])
+    bibtex_entry = ('@misc{' + url + ',\n\t'
```

Comment:
Use an [f-string](https://docs.python.org/3/reference/lexical_analysis.html#formatted-string-literals) instead of all the plusses